### PR TITLE
HW for day 2 : completed

### PR DESCRIPTION
**Pull request by gosophiego**

### CODE REVIEW

File: HW02/HW02_ch03_ex01.py [hunk 1]
```diff
@@ -14,20 +14,16 @@
 ################################################################################
 # Write your function below:
 # Body
-
-
-
-
-
-
+def right_justify (s):
```
Comment (danielsgriffin) [inline HW02/HW02_ch03_ex01.py]: remove the space between the function name and the parameters (for convention/readability)

File: HW02/HW02_ch05_ex00.py [hunk 2]
```diff
@@ -8,11 +8,10 @@
 # Write your functions below:
 # Body
 
-
-
-
-
-
+def do_n (f, n):
+	if (n > 0):
```
Comment (danielsgriffin) [inline HW02/HW02_ch05_ex00.py]: good. Not the logical temporal ordering difference between this and what some of your classmates did.

File: HW02/HW02_ch05_ex02.py [hunk 3]
```diff
@@ -21,10 +21,21 @@
 # Write your functions below:
 # Body
 
+def check_fermet (a, b, c, n):
+	if (a**n + b**n == c**n):
```
Comment (danielsgriffin) [inline HW02/HW02_ch05_ex02.py]: You are missing something

File: HW02/HW02_ch05_ex03.py [hunk 4]
```diff
@@ -23,10 +23,25 @@
 # Write your functions below:
 # Body
 
+def is_triangle (a, b, c):
+    if a > b + c or b > a + c or c > a + b:
```
Comment (danielsgriffin) [inline HW02/HW02_ch05_ex03.py]: For readability sake it is wise to use parenthesis here (I would also adjust some whitespace):
Ex.:
```
if (a > b+c) or (b > a+c) or (c > a+b):
```

**Comment by danielsgriffin** [comment]
Good job.

File: HW02/HW02_ch05_ex02.py (hunk 3 above)
Comment (gosophiego) [inline HW02/HW02_ch05_ex02.py]: if (a**n + b**n == c**n and n>2):

**Pull request closed without merging**